### DISCUSSION TITLE
send exposes of devices separately to iobroker

### DISCUSSION THREAD

**Discussion by mmaric55** [Koenkk/zigbee2mqtt#15379] [Devices]
hi,
not really sure how describe.
Currently sending values from temp and humidity sensor to iobroker.
I see the device name and the values but everything in one object.

Is it possible to send topic/device as folder and then the values as separated objects ?

![image](https://user-images.githubusercontent.com/119892636/205729434-d14ac409-5991-4bc0-b342-77c887e90d3d.png)

**Comment by mmaric55** [comment]
hi, i have solved it by my own. Thanks.

**Reply by WernerRH21** [reply]
Hi mmaric55,
actually I have the same problem. How did you solved it? Thanks

**Reply by mmaric55** [reply]
Hi, you need to change in settings, advanced from json to attribute.

![Screenshot_20221209-170616](https://user-images.githubusercontent.com/119892636/206743673-21234878-873f-4fff-8213-e79a5a12fe76.jpg)

**Reply by WernerRH21** [reply]
Thanks. That's it.
